keyboards/matrix/m12og/rev1/rev1.c

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2021 kb-elmo<mail@elmo.space>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "rev1.h"
+
+void board_init(void) {
+    writePinLow(A8);
+}
+
+void keyboard_post_init_user(void) {
```

Comment:
This should be the _kb version of the function or be prefaced with `__attribute__((weak))` so that users can replace functionality.